docs/Writing_tasks.md

```diff
@@ -57,58 +64,86 @@ const {
   install
 } = require('mrm-core');
 
+// 2 - Define task configuration.
+// Follows "enquirer" prompt definition.
+const parameters = {
+  eslintPreset: {
+    type: 'input',
+    message: 'ESLint preset to use as basis',
+    initial: 'eslint:recommended'
+  }
+};
+
 function task(config) {
-  // Task options
-  // mrm eslint --config:name pizza
-  const { name, eslintPreset } = config
-    .defaults({
-      // Default value
-      eslintPreset: 'eslint:recommended'
-    })
-    // Required option
-    .require('name')
-    .values();
-
-  // Use custom preset package from npm
+  // 3 - Load task configs (either from prompt or other methods).
+  const { eslintPreset } = config.values();
+
+  // 4 - Define packages to install.
   const packages = ['eslint'];
+
+  // 5 - Append custom eslint-config in case defined.
   if (eslintPreset !== 'eslint:recommended') {
     packages.push(`eslint-config-${eslintPreset}`);
   }
 
-  // Create or update .eslintignore
+  // 6 - Create or load .eslintignore, and set basic ignores.
   lines('.eslintignore')
     .add(['node_modules/'])
     .save();
 
-  // Read project’s package.json
+  // 7 - Create or load package.json.
   const pkg = packageJson();
 
   pkg
-    // Add lint script
+    // 8 - Set linting script.
     .setScript('lint', 'eslint . --cache --fix')
-    // Add pretest script
+    // 9 - Set pretest script.
     .prependScript('pretest', 'npm run lint')
+    // 10 - Save changes to package.json.
     .save();
 
-  // Read .eslintrc if it exists
+  // 11 - Create or load .eslintrc.
   const eslintrc = json('.eslintrc');
 
-  // Use Babel parser if the project depends on Babel
+  // 12 - Use Babel parser if the project depends on Babel.
   if (pkg.get('devDependencies.babel-core')) {
     const parser = 'babel-eslint';
     packages.push(parser);
     eslintrc.merge({ parser });
   }
 
-  // Set preset
-  eslintrc.set('extends', eslintPreset).save();
+  // 13 - Configure ESlint preset, if set (defaults to eslint:recommended).
+  if (eslintPreset) {
+    eslintrc.set('extends', eslintPreset);
+  }
+
+  // 14 - Save changes to .eslintrc.
+  eslintrc.save();
 
-  // Install npm dependencies
+  // 15 - Install new NPM dependencies.
   install(packages);
 }
 
+// 16 - Configure task static information (used by Mrm CLI).
+task.parameters = parameters;
 task.description = 'Adds ESLint';
+
 module.exports = task;
 ```
 
-There are more methods in `mrm-core` — check out [the docs](../packages/mrm-core#api) and [the default tasks](../Readme.md#tasks).
+> Have a look at [`mrm-core` docs](../packages/mrm-core#api) for more utility helpers, and [the default tasks](../Readme.md#tasks) for reference.
+
+### Configuration definition
```

Comment:
They call them _prompts_ in the docs, and I think it's much clearer than _definitons_.

https://github.com/enquirer/enquirer